--- omd/packages/glib/BUILD
+++ omd/packages/glib/BUILD
@@ -2,14 +2,14 @@
     name = "glibconfig",
     hdrs = select(
         {
-            "@//:lsb_filesystem_layout": ["lib/x86_64-linux-gnu/glib-2.0/include/glibconfig.h"],
+            "@//:lsb_filesystem_layout": ["lib/aarch64-linux-gnu/glib-2.0/include/glibconfig.h"],
             "@//:fhs_filesystem_layout": ["lib64/glib-2.0/include/glibconfig.h"],
         },
         no_match_error = "Please build with lsb or fhs filesystem layout",
     ),
     includes = select(
         {
-            "@//:lsb_filesystem_layout": ["lib/x86_64-linux-gnu/glib-2.0/include"],
+            "@//:lsb_filesystem_layout": ["lib/aarch64-linux-gnu/glib-2.0/include"],
             "@//:fhs_filesystem_layout": ["lib64/glib-2.0/include"],
         },
         no_match_error = "Please build with lsb or fhs filesystem layout",
@@ -20,7 +20,7 @@
     name = "libglib",
     shared_library = select(
         {
-            "@//:lsb_filesystem_layout": "lib/x86_64-linux-gnu/libglib-2.0.so",
+            "@//:lsb_filesystem_layout": "lib/aarch64-linux-gnu/libglib-2.0.so",
             "@//:fhs_filesystem_layout": "lib64/libglib-2.0.so",
         },
         no_match_error = "Please build with lsb or fhs filesystem layout",
--- omd/packages/libxml2/BUILD
+++ omd/packages/libxml2/BUILD
@@ -2,7 +2,7 @@
     name = "libxml2_shared",
     shared_library = select(
         {
-            "@//:lsb_filesystem_layout": "lib/x86_64-linux-gnu/libxml2.so",
+            "@//:lsb_filesystem_layout": "lib/aarch64-linux-gnu/libxml2.so",
             "@//:fhs_filesystem_layout": "lib64/libxml2.so",
         },
         no_match_error = "Please build with lsb or fhs filesystem layout",
--- omd/packages/rrdtool/BUILD.rrdtool.bazel
+++ omd/packages/rrdtool/BUILD.rrdtool.bazel
@@ -49,7 +49,7 @@
             "-I/usr/include/uuid",
             "-I/usr/lib64/glib-2.0",
             "-I/usr/lib64/glib-2.0/include",
-            "-I/usr/lib/x86_64-linux-gnu/glib-2.0/include",
+            "-I/usr/lib/aarch64-linux-gnu/glib-2.0/include",
         ]) + "'",
     ],
     copts = [
