--- omd/packages/snap7/snap7_http.bzl
+++ omd/packages/snap7/snap7_http.bzl
@@ -12,7 +12,7 @@
             # "https://sourceforge.net/projects/snap7/files/" + version_str + "snap7-full-" + version_str + ".7z",
             UPSTREAM_MIRROR_URL + filename,
         ],
-        sha256 = "fe137737b432d95553ebe5d5f956f0574c6a80c0aeab7a5262fb36b535df3cf4",
+        sha256 = "aa675dfc77a057d99f08254a14217eb1728bde663324c2e622fd1b17f88bf12e",
         build_file = "@omd_packages//omd/packages/snap7:BUILD.snap7.bazel",
         strip_prefix = "snap7-" + version_str,
     )
