--- omd/packages/Python/Python.make
+++ omd/packages/Python/Python.make
@@ -7,7 +7,7 @@
 PYTHON_BUILD := $(BUILD_HELPER_DIR)/$(PYTHON_DIR)-build
 PYTHON_INTERMEDIATE_INSTALL := $(BUILD_HELPER_DIR)/$(PYTHON_DIR)-install-intermediate
 PYTHON_INSTALL := $(BUILD_HELPER_DIR)/$(PYTHON_DIR)-install
-PYTHON_SYSCONFIGDATA := _sysconfigdata__linux_x86_64-linux-gnu.py
+PYTHON_SYSCONFIGDATA := _sysconfigdata__linux_aarch64-linux-gnu.py
 
 PYTHON_INSTALL_DIR := $(INTERMEDIATE_INSTALL_BASE)/$(PYTHON_DIR)
 
--- omd/packages/Python/BUILD.Python.bazel
+++ omd/packages/Python/BUILD.Python.bazel
@@ -64,7 +64,7 @@
     postfix_script = """
         py_install_dir="$BUILD_TMPDIR/{installPrefix}";
         bin_dir="$INSTALLDIR/bin";
-        sysconfig_path="$INSTALLDIR/lib/python{pyMajMin}/_sysconfigdata__linux_x86_64-linux-gnu.py"
+        sysconfig_path="$INSTALLDIR/lib/python{pyMajMin}/_sysconfigdata__linux_aarch64-linux-gnu.py"
         # omd_prefix must be fixed-up to the corresponding omd path (e.g. /omd/versions/2.2.0p2.cee)
         omd_prefix="/replace-me"
 
