--- Makefile    2024-07-13 10:52:19.930691384 +0000
+++ Makefile_v2 2024-07-13 11:43:16.214539668 +0000
@@ -426,7 +426,7 @@
        @( \
                echo "Locking Python requirements..." ; \
                flock $(LOCK_FD); \
-               ( SKIP_MAKEFILE_CALL=1 $(PIPENV) lock --python $(PYTHON_MAJOR_DOT_MINOR) ) || ( $(RM) -r .venv ; exit 1 ) \
+               ( SKIP_MAKEFILE_CALL=1 $(PIPENV) lock --verbose --python $(PYTHON_MAJOR_DOT_MINOR) ) || ( $(RM) -r .venv ; exit 1 ) \
        ) $(LOCK_FD)>$(LOCK_PATH); \

 # Remake .venv everytime Pipfile or Pipfile.lock are updated. Using the 'sync'
@@ -446,5 +446,5 @@
              echo "Cleaning up .venv before sync..."; \
              $(RM) -r .venv; \
            fi; \
-           ( SKIP_MAKEFILE_CALL=1 SETUPTOOLS_ENABLE_FEATURES="legacy-editable" VIRTUAL_ENV="" $(PIPENV) sync --python $(PYTHON_MAJOR_DOT_MINOR) --dev && touch .venv ) || ( $(RM) -r .venv ; exit 1 ) \
+           ( SKIP_MAKEFILE_CALL=1 SETUPTOOLS_ENABLE_FEATURES="legacy-editable" VIRTUAL_ENV="" $(PIPENV) sync --verbose --python $(PYTHON_MAJOR_DOT_MINOR) --dev && touch .venv ) || ( $(RM) -r .venv ; exit 1 ) \
        ) $(LOCK_FD)>$(LOCK_PATH)
