--- Makefile    2024-07-13 10:32:29.335335063 +0000
+++ Makefile_v2 2024-07-13 10:35:28.037187272 +0000
@@ -333,7 +333,7 @@
            pip \
            pipenv=="$(PIPENV_VERSION)" \
            virtualenv=="$(VIRTUALENV_VERSION)" \
-           wheel
+           wheel --break-system-packages
        if ! type rustup >/dev/null 2>&1; then \
                curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh; \
                source $$HOME/.cargo/env; \