--- omd/packages/net-snmp/BUILD.net-snmp.bazel
+++ omd/packages/net-snmp/BUILD.net-snmp.bazel
@@ -172,7 +172,7 @@
             make -C perl DESTDIR=$${INSTALLDIR}/ \
                 INSTALLSITEARCH=/lib/perl5/lib/perl5 \
                 INSTALLSITEMAN3DIR=/share/man/man3 \
-                INSTALLARCHLIB=/lib/perl5/lib/perl5/x86_64-linux-gnu-thread-multi \
+                INSTALLARCHLIB=/lib/perl5/lib/perl5/aarch64-linux-gnu-thread-multi \
                 install
 
         chmod u+w -R $${INSTALLDIR}
