--- bazel/thirdparty/modules/perl-modules/0.0.1.cmk.5/src/lib/BuildHelper.pm
+++ bazel/thirdparty/modules/perl-modules/0.0.1.cmk.5/src/lib/BuildHelper.pm
@@ -645,7 +645,7 @@
     if($file =~ m/\.zip$/gmx) {
         cmd("unzip $file");
     } else {
-        cmd("tar zxf $file");
+        cmd("tar zxf $file --no-same-owner");
     }
     my $dir = $file;
     $dir =~  s/(\.tar\.gz|\.tgz|\.zip)//g;
