--- MODULE.bazel
+++ MODULE.bazel
@@ -380,10 +380,10 @@
 
 http_file(
     name = "jaeger",
-    sha256 = "1d810bf04f7e08dac796ad532fc68f1a8fceb0333d2524ae767d88533e69ed75",
+    sha256 = "a4c245e1e928ce16e89ae995cecf6cda748d24a8fff6e7ca84758552563a04e5",
     urls = [
-        "https://github.com/jaegertracing/jaeger/releases/download/v2.18.0/jaeger-2.18.0-linux-amd64.tar.gz",
-        UPSTREAM_MIRROR_URL + "jaeger-2.18.0-linux-amd64.tar.gz",
+        "https://github.com/jaegertracing/jaeger/releases/download/v2.18.0/jaeger-2.18.0-linux-arm64.tar.gz",
+        UPSTREAM_MIRROR_URL + "jaeger-2.18.0-linux-arm64.tar.gz",
     ],
 )
 
--- omd/packages/jaeger/BUILD
+++ omd/packages/jaeger/BUILD
@@ -8,7 +8,7 @@
 genrule(
     name = "fetch_package",
     srcs = ["@jaeger//file"],
-    outs = ["jaeger-" + JAEGER_VERSION + "-linux-amd64.tar.gz"],
+    outs = ["jaeger-" + JAEGER_VERSION + "-linux-arm64.tar.gz"],
     cmd = """
         cp $(SRCS) $(OUTS)
     """,
@@ -25,7 +25,7 @@
 	    --strip-components=1 \
 	    -C $${target_path} \
 	    -f $(SRCS) \
-	    jaeger-%s-linux-amd64/jaeger
+	    jaeger-%s-linux-arm64/jaeger
         strip -s $@ -o $@.tmp
         mv $@.tmp $@
     """ % JAEGER_VERSION,
